content/02.main-text.md

```diff
@@ -440,3 +440,23 @@ Source code for Manubot resides in the following repositories:
 
 + <https://github.com/manubot/manubot> ([GitLab mirror](https://gitlab.com/manubot/manubot), archived at [`swh:1:dir:1eb3b7e6d7e21239fedccf25a186af622ee6912e`](https://archive.softwareheritage.org/swh:1:dir:1eb3b7e6d7e21239fedccf25a186af622ee6912e;origin=https://github.com/manubot/manubot.git/), packaged on [PyPI](https://pypi.org/project/manubot/))
 + <https://github.com/manubot/rootstock> ([GitLab mirror](https://gitlab.com/manubot/rootstock), archived at [`swh:1:dir:cfc3af2e8e1a0d9b639fdab0943731e608910731`](https://archive.softwareheritage.org/swh:1:dir:cfc3af2e8e1a0d9b639fdab0943731e608910731;origin=https://github.com/manubot/rootstock.git/)).
+
+## Supporting Information
+
+<div id="vid:usage">
```

Comment:
Noting that the recently released Pandoc 2.7.3 may have allow a more markdown-like syntax for video embeds. See https://twitter.com/dhimmel/status/1148978224996556800.